### PR TITLE
Adding travis

### PR DESCRIPTION
**Pull request by mortonjt**

### CODE REVIEW

**Comment by mortonjt** [comment]
@jwdebelius want to merge this?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  - wget http://repo.continuum.io/miniconda/Miniconda3-3.7.3-Linux-x86_64.sh -O miniconda.sh
+  - chmod +x miniconda.sh
+  - ./miniconda.sh -b
+  - export PATH=/home/travis/miniconda3/bin:$PATH
+  # Update conda itself
+  - conda update --yes conda
+install:
+  - conda create --yes -n env_name python=$PYTHON_VERSION pip numpy scipy nose flake8 h5py
+  - source activate env_name
+  - conda install --yes numpy scikit-bio jupyter seaborn matplotlib=1.5.1
```
Comment (jwdebelius) [inline .travis.yml]: Does it matter that skbio 0.5.0 isn't in the general conda?

**Comment by mortonjt** [comment]
@jwdebelius I addressed your comment.  Should be good to go.

**Review by jwdebelius — APPROVED** [review]
Thank you!

**Comment by jwdebelius** [comment]
:+1:

**Pull request merged**